tests/test_visitors/test_tokenize/test_primitives/test_string_tokens/test_mulitiline_formatted_string.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    MultilineFormattedStringViolation,
+)
+from wemake_python_styleguide.visitors.tokenize.primitives import (
+    MultilineFormattedStringTokenVisitor,
+)
+
+single_quote_formatted_string_wrong = """x=f'{ 1
```

Comment:
Please, separete `# Correct:` and `# Wrong:` cases by comments. It would be easier to read